src/ol/renderer/maprenderer.js

```diff
@@ -9,7 +9,6 @@ goog.require('ol.events.EventType');
 goog.require('ol.extent');
 goog.require('ol.functions');
 goog.require('ol.layer.Layer');
-goog.require('ol.renderer.Layer');
```

Comment:
We'll need to leave this one in since`ol.layer.Layer.visibleAtResolution()` is used [in this file](https://github.com/probins/ol3/blob/52afa50eec58a3a0c86442687725ab9343619afb/src/ol/renderer/maprenderer.js#L206).